service/main.go

```diff
@@ -43,6 +48,15 @@ func MainContext(ctx context.Context) {
 	params := otelcol.CollectorSettings{
 		Factories: components,
 		BuildInfo: info,
+		ConfigProviderSettings: otelcol.ConfigProviderSettings{
+			ResolverSettings: confmap.ResolverSettings{
+				ProviderFactories: []confmap.ProviderFactory{
+					fileprovider.NewFactory(),
+					envprovider.NewFactory(),
+					yamlprovider.NewFactory(),
```

Comment:
According to https://github.com/open-telemetry/opentelemetry-collector/issues/10290#issuecomment-2145370916 the default should be `"env"` (or in this case `envprovider`). 

Spoke offline with @franciscovalentecastro, and integration tests fail without the file provider.

Could there be other providers that we are might be missing that are tests are not picking up?

Reply:
As discussed offline, a guideline to set providers here could be the `default` that was set before the update. I found `newDefaultConfigProviderSettings` was modified here https://github.com/open-telemetry/opentelemetry-collector/pull/10436. I set the defaults from there :

```
func newDefaultConfigProviderSettings(uris []string) ConfigProviderSettings {
	return ConfigProviderSettings{
		ResolverSettings: confmap.ResolverSettings{
			URIs: uris,
			ProviderFactories: []confmap.ProviderFactory{
				fileprovider.NewFactory(),
				envprovider.NewFactory(),
				yamlprovider.NewFactory(),
				httpprovider.NewFactory(),
				httpsprovider.NewFactory(),
			},
			ConverterFactories: []confmap.ConverterFactory{expandconverter.NewFactory()},
		},
	}
}
```